app/code/Magento/Sales/Model/Service/CreditmemoService.php

```diff
@@ -104,18 +104,6 @@ public function __construct(
     public function cancel($id)
```

Comment:
Some static tests fails because with your refactoring the method parameter `$id` is not used any more.
Please, add a `@SuppressWarnings(PHPMD.UnusedFormalParameter)` directive in method's docblock to avoid the static test to fail.

Reply:
Thanks. Fixed it.